esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Oclim.json

```diff
@@ -9,7 +9,7 @@
         "int_missing_value": "-999", 
         "product": "model-output", 
         "approx_interval": "30.00000", 
-        "generic_levels": "olevel", 
+        "generic_levels": "olevel olevhalf", 
```

Comment:
can anyone check this is not in any of the issues on the pcmdi github page pls? @bouweandela @zklaus @jvegasbsc

Reply:
https://github.com/PCMDI/cmor/issues/559